doc/man7/EVP_MD-MD5-SHA1.pod

```diff
@@ -0,0 +1,56 @@
+=pod
+
+=head1 NAME
+
+EVP_MD-MD5-SHA1 - The MD5-SHA1 EVP_MD implementation
+
+=head1 DESCRIPTION
+
+Support for computing MD5-SHA1 digests through the B<EVP_MD> API.
+
+MD5-SHA1 is a rather special digest that's used with SSL v3.
```

Comment:
I know it’s a matter of taste, but I‘d avoid colloquial abbreviations (that’s) in man pages. 

Also: SSLv3 instead of SSL v3?

Reply:
Fixed